repos/fdbt-netex-output/src/netex-convertor/period-tickets/periodTicketNetexHelpers.ts

```diff
@@ -875,6 +930,60 @@ export const getPeriodAvailabilityElement = (
     },
 });
 
+export const getExemptionsElement = (
+    id: string,
+    validityParametersObject: {},
+    hasTimeRestriction: boolean,
+    productName?: string,
```

Comment:
how come productName and groupsOfLinesRef are optional?

Reply:
done in other functions such as getLines list it impacts other netex if I remove it

Reply:
done in other functions such as getLines list it impacts other netex if I remove it